cassandra-migration/src/main/java/org/cognitor/cassandra/migration/Database.java

```diff
@@ -31,13 +32,13 @@
      * Insert statement that logs a migration into the schema_migration table.
      */
     private static final String INSERT_MIGRATION = "insert into %s"
-            + "(applied_successful, version, script_name, script, executed_at) values(?, ?, ?, ?, ?)";
+            + "(applied_successful, version, script_name, script, checksum, executed_at) values(?, ?, ?, ?, ?, ?)";
 
     /**
      * Statement used to create the table that manages the migrations.
      */
     private static final String CREATE_MIGRATION_CF = "CREATE TABLE %s"
-            + " (applied_successful boolean, version int, script_name varchar, script text,"
+            + " (applied_successful boolean, version int, script_name varchar, script text, checksum int,"
```

Comment:
Here there needs to be something more done as this would not work for everybody who already has this table.